src/app/Fake.IO.FileSystem/ChangeWatcher.fs

```diff
@@ -0,0 +1,112 @@
+[<AutoOpen>]
```

Comment:
We no longer use AutoOpen for FAKE 5 APIs.

We try to have nice APIs for the Call side (IE: `ChangeWatcher.Run` or `ChangeWatcher.RunWithOptions`)